media-libs/x265/x265-3.4.ebuild

```diff
@@ -6,15 +6,16 @@ EAPI=7
 inherit cmake multilib-minimal multilib multibuild flag-o-matic
 
 if [[ ${PV} = 9999* ]]; then
-	inherit mercurial
-	EHG_REPO_URI="https://bitbucket.org/multicoreware/x265"
+	inherit git-r3
+	EGIT_REPO_URI="https://bitbucket.org/multicoreware/x265_git/"
+	S=${WORKDIR}/${P}/source
 else
-	SRC_URI="https://bitbucket.org/multicoreware/x265/downloads/${PN}_${PV}.tar.gz"
+	SRC_URI="https://bitbucket.org/multicoreware/x265_git/get/${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Official repo of x265 remains be as a mercurual repository located here: http://hg.videolan.org/x265